### ISSUE TITLE
Training a model using a pre-trained model

### ISSUE THREAD

**Issue by aloofhawk** [fchollet/keras#6027]
I have a pre-trained model in Keras. I would like to train another model where the output of the model is input for the already trained model and the output of the already trained model is being used in the loss function of the untrained model. Something like 

    in_a + mod_b(some kind of feedback from B here) --> Model A --> out_a+something added --> Model B --> out_b
    
    error  = (in_a - out_b)**2

and then use this error to train Model A.
`in_a` can be treated as a constant in this system and there is a feedback loop as well

I was thinking of reassigning the loss function of A as the error above. Can somebody tell me if this is possible or not or if there is a better way of doing this. I had asked this question on stack overflow but didnt get a very satisfactory answer

More specifically it is a control problem so if someone just tell me how to do a model reference control in keras that would be pretty great too
